section4/exercises/classMethods.js

```diff
@@ -28,13 +28,37 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
-    this.protein = protein;
+  constructor(protein, base, toppings)  {    //parameters
+    this.protein = protein; //Properties
     this.base = base;
+    this.toppings = toppings;
+
   }
 
+changeProtein() {
+  this.protein = "Sub meat for Chicharron";
+}
+
+addTopping() {
```

Comment:
This `addTopping` should take one argument and push that argument into the `this.toppings` array -

```
addTopping(newTopping) {
     this.toppings.push(newTopping);
}
```